gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -188,8 +189,24 @@ def consumer_ptg_removed_notification(self, context, network_function_id,
         service_orchestrator.handle_consumer_ptg_removed(
             context, network_function_id, policy_target_group)
 
+    @log_helpers.log_method_call
+    def get_network_function_details(self, context, network_function_id):
+        '''Invoked in an RPC Call.
+
+        Return the Network function Details object'''
+        service_orchestrator = ServiceOrchestrator(self._controller, self.conf)
+        return service_orchestrator.get_network_function_details(
+            network_function_id)
```

Comment:
We already have get_network_functions() function in same class, why we are adding 1 more API for same.